pkg/arvo/lib/push-hook.hoon

```diff
@@ -1,15 +1,55 @@
+::  lib/push-hook/hoon: helper for creating a push hook
+::  
+::   lib/push-hook is a helper for automatically pushing data from a
+::   local store to the corresponding pull-hook on remote ships. It also
+::   proxies remote pokes to the store.
+::
+::   ## Interfacing notes:
+::
+::   lib/push-hook provides the inner door with its own separate state.
+::   The inner door may interact with the library by producing cards or
+::   scrying. Do not pass any cards on a wire beginning with /helper as
```

Comment:
There are no special scries here, either. But at least it passes `+on-peek` calls through to `og` properly!